src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
+
+  type Key = Array[Byte]
+  type Value = Array[Byte]
```

Comment:
Could you add a few more words in the scaladoc explaining why a `Key`s and `Value`s are `Array[Byte]`s and not `ByteString`s or any other type?